x-pack/plugins/apm/public/components/app/correlations/failed_transactions_correlations.tsx

```diff
@@ -456,54 +425,55 @@ export function FailedTransactionsCorrelations({
       <EuiSpacer size="s" />
 
       <CorrelationsProgressControls
-        progress={progress}
-        isRunning={isRunning}
-        onRefresh={startFetchHandler}
+        progress={progressNormalized}
+        isRunning={progress.isRunning}
+        onRefresh={startFetch}
         onCancel={cancelFetch}
       />
 
-      {ccsWarning && (
+      {response.ccsWarning && (
         <>
           <EuiSpacer size="m" />
+          {/* Failed transactions correlations uses ES aggs that are available since 7.15 */}
           <CrossClusterSearchCompatibilityWarning version="7.15" />
         </>
       )}
 
-      {inspectEnabled &&
-      selectedTerm?.pValue != null &&
-      (isRunning || correlationTerms.length > 0) ? (
-        <>
-          <EuiSpacer size="m" />
-          <Summary
-            items={[
-              <EuiBadge color="hollow">
-                {`${selectedTerm.fieldName}: ${selectedTerm.key}`}
-              </EuiBadge>,
-              <>{`p-value: ${selectedTerm.pValue.toPrecision(3)}`}</>,
-            ]}
-          />
-        </>
-      ) : null}
+      {showSummaryBadge &&
+        selectedTerm !== undefined &&
+        selectedTerm.pValue !== null && (
```

Comment:
Do we care about the difference between `undefined` and `null`. If not:
```suggestion
        selectedTerm?.pValue && (
```